### DISCUSSION TITLE
Content and i18n - Document not found for non-default locales

### DISCUSSION THREAD

**Discussion by vlamic** [nuxt/content#2595] [Q&A]
Hi, 

I'm trying to setup a project which is driven by content and supports multiple languages, so I have created two dirs in my `content` folder: `en` and `es`.  When I try to access the default routes (e.g.  http://localhost:3000/ or http://localhost:3000/about), it loads  the pages correctly. However, when I try to access non-default pages, it's not able to locate the content (e.g. http://localhost:3000/en/about or http://localhost:3000/en). Am I missing something?

I have uploaded a sample project here: https://github.com/vlamic/content-app-test

Content structure:

![iScreen Shoter - Code - 240325120127](https://github.com/nuxt/content/assets/1814198/a2a11368-c02f-4afd-a4a5-b959e96f7941)

Pages structure:

![iScreen Shoter - Code - 240325120213](https://github.com/nuxt/content/assets/1814198/ac906c90-4185-4d0e-b364-59b9c3b190ea)

Nuxt config:

![iScreen Shoter - Code - 240325120230](https://github.com/nuxt/content/assets/1814198/9fa87ba1-fdd5-4944-9644-fe2a52cf9774)

Thank you!

**Comment by RollingTL** [comment]
I have similar issue. More details:
https://github.com/nuxt/nuxt/discussions/26699
